index.bs

```diff
@@ -56,10 +56,12 @@ Boilerplate: omit conformance
 <pre class="link-defaults">
 spec:infra; type:dfn; for:set; text:for each
 spec:infra; type:dfn; for:/; text:set
+spec:infra; type:dfn; text:implementation-defined
 spec:streams; type:interface; text:ReadableStream
 spec:fetch; type:dfn; for:/; text:fetch
 spec:url; type:dfn; text:scheme
 spec:url; type:dfn; text:fragment
+spec:webdriver-bidi; type:dfn; text:session
```

Comment:
I needed to add this to make bikshed work happy, but it seems to have nothing to do with my PR and it seems wrong.

Reply:
I think you need to replace all instances of `[=session=]` with `[=WebTransport session=]` in the PR.

Reply:
Yes, that looks like it was it.  Thank you.